test/tests/mysql-check-config/run.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+serverImage="$("$dir/../image-name.sh" librarytest/mysql-initdb "$image")"
+"$dir/../docker-build.sh" "$dir" "$serverImage" <<EOD
+FROM $image
+RUN echo '!includedir /tmp/conf.d/' >> \$(ls /etc/mysql/my.cnf /etc/my.cnf | head -1)
+EOD
+
+export MYSQL_ROOT_PASSWORD='this is an example test password'
+export MYSQL_USER='0123456789012345' # "ERROR: 1470  String 'my cool mysql user' is too long for user name (should be no longer than 16)"
+export MYSQL_PASSWORD='my cool mysql password'
+export MYSQL_DATABASE='my cool mysql database'
+
+cname="mysql-container-$RANDOM-$RANDOM"
+cid="$(
+	docker run -d \
+		-e MYSQL_ROOT_PASSWORD \
+		-e MYSQL_USER \
+		-e MYSQL_PASSWORD \
+		-e MYSQL_DATABASE \
+		-v "$dir:/tmp/conf.d" \
+		--name "$cname" \
+		"$serverImage"
+)"
+trap "docker rm -vf $cid > /dev/null" EXIT
+
+while docker ps -f "name=$cname" | grep -q "$cname"; do
+	sleep 1
+done
+
+docker logs "$cname" 2>&1 \
+	| grep -q 'ERROR: mysqld failed while attempting to check config'
```

Comment:
This test was added for cover `_check_config` function (from entrypoint) by tests.
So, the goal is sure that these lines https://github.com/docker-library/mysql/blob/master/5.7/docker-entrypoint.sh#L61-L66 was run in fact.
`_check_config` function print `ERROR: mysqld failed while attempting to check config` error in case of any detected issues in config.
I am ok to don't use `grep`, if you see any possible way which will allow us to cover this function by tests without grep - just give me know!